tests/lib/utils/utils-test.js

```diff
@@ -94,6 +94,14 @@ describe('isArrowFunctionExpression', () => {
   });
 });
 
+describe('isConciseArrowFunctionExpressionWithCall', () => {
+  const node = parse('test = () => foo()').right;
+
+  it('should check if node is concise arrow function expression with call expression body', () => {
+    assert.ok(utils.isConciseArrowFunctionWithCallExpression(node));
+  });
```

Comment:
In this case it might have a sense indeed. Just to be sure that arrow function with body is not filling requirements of this function. Could you please address that as well @sudowork ? :)